lib/order.rb

```diff
@@ -0,0 +1,36 @@
+require_relative 'sms'
+
+class Order
+  attr_reader :current_order, :order_total
+
+  def initialize(sms_klass = Sms.new)
+    @sms = sms_klass
+    @current_order = []
+    @order_total = 0
+    @order_number = 0
+  end
+
+  def set_current_order(dish, quantity, price)
+    @current_order << {dish: dish, quantity: quantity, price: price}
+    @order_total += @current_order.last[:price]
+  end
+
+  def sum_up_prices
+    current_order.map {|order| order[:price]}.reduce(:+)
+  end
+
+  def assign_order_number
+    @order_number = rand(5000)
+  end
+
+  def wrong_total?
+    !!(sum_up_prices != order_total)
```

Comment:
Style/DoubleNegation: Avoid the use of double negation (`!!`). (https://github.com/bbatsov/ruby-style-guide#no-bang-bang)